Odata-docs/performance/using-utf8jsonwriter.md

```diff
@@ -0,0 +1,22 @@
+---
+title:  "Use Utf8JsonWriter"
+description: A guide on how to improve performance using Utf8JsonWriter.
+date:   2024-01-23
+ms.date: 1/23/2024
+author: habbes
+ms.author: clhabins
+---
+
+# Use `Utf8JsonWriter`
+
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v8.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)][!INCLUDE[appliesto-odatalib](../includes/appliesto-odatalib-v7.md)]
+
+The OData Core library implements a wrapper on top of the `Utf8JsonWriter` as an alternative to the default JSON writer. This writer offers better performance and fewer memory allocations compared to using the default JSON writer used in `Microsoft.OData.Core`.
```

Comment:
```suggestion
The OData core library implements a wrapper on top of the `Utf8JsonWriter` to complement the built-in JSON writer. This writer offers better performance and fewer memory allocations compared to the built-in JSON writer that's used by default in `Microsoft.OData.Core`.
```

Reply:
Regarding the use of "built-in", could that be misleading given that both implementations of `IJsonWriter` are technically built-in to the core library?